src/RoutingList/hooks/useRoutingListById/useRoutingListById.js

```diff
@@ -0,0 +1,37 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import { ROUTING_LIST_API } from '../../../common/constants';
+
+const DEFAULT_ROUTING_LIST = {
+  name: '',
+  notes: '',
+  userIds: [],
+  users: [],
+};
+
+export const useRoutingListById = (listingId) => {
```

Comment:
what is listingId? why not just id or listId or routingListId?


just `useRoutingList`